sympy/functions/special/tests/test_spec_polynomials.py

```diff
@@ -107,6 +106,21 @@ def test_legendre():
     assert legendre(10, 0) != 0
     assert legendre(11, 0) == 0
 
+    assert legendre(-1, x) == 1
```

Comment:
It is probably not necessary to repeat *all* of the tests for positive `n` with negative `-n - 1`. A few could be chosen.

Reply:
done :+1: